.vscode/extensions.json

```diff
@@ -1,9 +0,0 @@
-{
-	// See http://go.microsoft.com/fwlink/?LinkId=827846
```

Comment:
This will remove the pretty highlighting of the scenarios right. Why?

Reply:
does it? and if it should done in an other way, because .vscode are IDE files, they should not include in github. Its also in the gitignore so local changes are keeped

Reply:
good point, let's see